.snyk

```diff
@@ -0,0 +1,10 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.19.0
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JS-UNDERSCORE-1080984:
```

Comment:
I checked and the "underscore" package is only used by the playground which is a dev tool and not used in prod. https://github.com/OriginProtocol/origin-dollar/blob/dc9ec97fadb90b409be0f7749be97a9b149d1855/playground/src/stores.js#L4

Seems fine to ignore. Or better we could upgrade without too much worries of breaking something in prod.

If you decide to ignore, it would be useful to document the reason in this .snyk file so that we don't have to look it up next time we wonder about this ignore...

Reply:
snyk ignored - `dev dependencies` so more than likely this module is in the wrong place. I ignored this one really just to generate the `.snyk` policy file, but you're right I should have put a description